### PR TITLE
feat: mobile rename profile

### LINKED ISSUE #2511: [Task]: Rename account on mobile
### Task description

The main menu has an option to rename Profile, where the folder is renamed. Will use the Capacitor plugin `SecureFilesystemAccess` for that feature.
Currently on the Capacitor Bindings plugin we haven't an option to get the profile folder name and create it, we must do this too.

### Requirements

- [x] Implement `renameProfileFolder` on `CapacitorApi.ts`
- [x] Implement create a specific folder for each profile on Capacitor Bindings plugin for both Android & iOS
- [x] Verify the folder was renamed on device filesystem on iOS
- [x] Verify the folder was renamed on device filesystem on Android

### Acceptance criteria

Upon the folder has renamed the profile will be changed, so we should assume it's done when the profile is successfully renamed in the UI.

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by amadeu2**
## Summary
The main menu has an option to rename Profile where the folder is renamed. Will use the Capacitor plugin SecureFilesystemAccess for that feature.

### Changelog
-[x] Implement renameProfileFolder on CapacitorApi.ts
-[x] Verify the folder was renamed on device filesystem

## Relevant Issues
Closes #2511

## Type of Change
Please select any type below that is applicable to your changes, and delete those that are not.
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [ ] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [ ] __Fix__ - a change which fixes an issue
- [x] __New__ - a change which implements a new feature
- [ ] __Update__ - a change which updates existing functionality

### Platforms
Please select any platforms where your changes have been tested.
- __Desktop__
	- [ ] MacOS
	- [ ] Linux
	- [ ] Windows
- __Mobile__
	- [x] iOS
	- [x] Android

### Instructions
Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

## Checklist
Please tick all of the following boxes that are relevant to your changes, and delete those that are not.
- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/backend/bindings/capacitor/android/src/main/java/org/iota/walletactorsystem/WalletPlugin.java [hunk 1]
```diff
@@ -23,13 +23,16 @@ public void load() {
 
     @PluginMethod()
     public void initialize(final PluginCall call) {
-        if (isInitialized) return;
-        if (!call.getData().has("actorId")) {
-            call.reject("actorId is required");
-            return;
+        if (isInitialized) {
+            call.reject("Wallet is yet initialized!");
```
Comment (rajivshah3) [inline packages/backend/bindings/capacitor/android/src/main/java/org/iota/walletactorsystem/WalletPlugin.java:27]:
```suggestion
            call.reject("Wallet is already initialized!");
```

File: packages/backend/bindings/capacitor/ios/Plugin/Plugin.swift [hunk 2]
```diff
@@ -10,12 +10,13 @@ public class WalletPlugin: CAPPlugin {
     @objc func initialize(_ call: CAPPluginCall) {
         do {
             guard !isInitialized else { return }
-            guard let actorId = call.getString("actorId") else {
-                return call.reject("actorId is required")
+            guard let actorId = call.getString("actorId"),
+                let storagePath = call.getString("storagePath")  else {
+                return call.reject("actorId adn storagePath is required")
```
Comment (rajivshah3) [inline packages/backend/bindings/capacitor/ios/Plugin/Plugin.swift:15]:
```suggestion
                return call.reject("actorId and storagePath is required")
```

Comment (amadeu2) [reply packages/backend/bindings/capacitor/ios/Plugin/Plugin.swift:15]: thanks!

**Review by amadeu2 — COMMENTED** [review]
checked again, seems ready 🙂

File: packages/backend/bindings/capacitor/ios/Plugin/Plugin.swift [hunk 3]
```diff
@@ -10,12 +10,13 @@ public class WalletPlugin: CAPPlugin {
     @objc func initialize(_ call: CAPPluginCall) {
         do {
             guard !isInitialized else { return }
-            guard let actorId = call.getString("actorId") else {
-                return call.reject("actorId is required")
+            guard let actorId = call.getString("actorId"),
+                let storagePath = call.getString("storagePath")  else {
```
Comment (rajivshah3) [inline packages/backend/bindings/capacitor/ios/Plugin/Plugin.swift:14]:
```suggestion
                let storagePath = call.getString("storagePath") else {
```

File: packages/backend/bindings/capacitor/ios/Plugin/Plugin.swift [hunk 4]
```diff
@@ -10,12 +10,13 @@ public class WalletPlugin: CAPPlugin {
...
+                return call.reject("actorId and storagePath is required")
```
Comment (rajivshah3) [inline packages/backend/bindings/capacitor/ios/Plugin/Plugin.swift:15]:
```suggestion
                return call.reject("actorId and storagePath are required")
```

**Review by rajivshah3 — APPROVED** [review]

**Pull request merged**